### ISSUE TITLE
VoiceOver screen reader does not announce certain formatting in the rendered view

### ISSUE THREAD

**Issue by zanesims** [schuyler/macdown3000#545]
For blind users using the native MacOS VoiceOver screen reader, the rendered web view does not report certain text formatting attributes. This applies for `**bold**, *italics*, and <u>underline</u>. That is true whether reading a whole block of text at once containing some of that formatting, or if the VoiceOver cursor is placed within a word formatted that way and VO+T is pressed. ~~strikethrough~~ reads "deletion" before the text it is applied to if VoiceOver is reading the whole block of text it is in, but if the VoiceOver cursor is positioned in just the text with the strikethrough and VO+T is used to read the text formatting, it does not report the strikethrough. Interestingly, headers are read as "header level X" and also "bold" is read aloud by VoiceOver.`

At a minimum it should be reading the attributes as it comes across it in the block of text (when VoiceOver is configured to do so), and it should read the attributes when the VoiceOver cursor is on a piece of text with those formatting attributes and VO+T is pressed. Some apps also add the aditional feature of stopping the VO cursor when it encounters one of these attributes so the user has to advance the cursor manually and that helps the formatting stand out even more. So something like: "This is a block of (pause...VO+-> is pressed to continue) bold text for (pause...VO+-> is pressed to continue again) sake of example." In that example, the "bold text for" is some bold text read from the block.

I'm using Macdown 3000.0.7 (1429) on an M1 MacBook Air (2020) running MacOS 26.5.1

**Comment by schuyler** [comment]
@zanesims thanks for this report. Accessbility is really important, but also candidly I don't have a lot of experience with VoiceOver and I'm going to need some time to figure out how to text any improvements we make around this. Thanks for your patience.
